util/linux_install.sh

```diff
@@ -126,6 +140,25 @@ elif grep ID /etc/os-release | grep -qE "opensuse|tumbleweed"; then
 		wget \
 		zip
 
+elif grep ID /etc/os-release | grep -q slackware; then
+	echo $SLACKWARE_WARNING
```

Comment:
```suggestion
	printf "$SLACKWARE_WARNING"
```
The quotes are important. Again, you can use _Apply suggestion_ to commit this directly.